Cargo.toml

```diff
@@ -19,7 +19,7 @@ targets = [
 ]
 
 [dependencies]
-libc = "0.2.20"
+libc = { git = "https://github.com/rust-lang/libc.git", rev = "d6fe782", features = [ "extra_traits" ] }
```

Comment:
I'd prefer to wait for the libc version that contains this to be released instead of using the git commit.